legal-api/src/legal_api/resources/v2/business/colin_sync.py

```diff
@@ -289,6 +290,9 @@ def _set_shares(primary_or_holding_business, amalgamation_filing, transaction_id
     amalgamation_filing['shareStructure'] = {'shareClasses': share_classes}
     business_dates = [item.resolution_date.isoformat() for item in primary_or_holding_business.resolutions]
     if business_dates:
+        formatted_dates = [
+            datetime.strptime(date, '%Y-%m-%d').strftime('%B %-d, %Y').replace(' 0', ' ') for date in business_dates
+        ]
```

Comment:
No need to format here. This is for colin sync

Reply:
Sure, just restored